docs/github/v2/06-remote-builder.md

```diff
@@ -0,0 +1,123 @@
+# Remote builder
+
+## What is it?
+
+GitHub Actions run on build machines provided by GitHub. You can see info about the specifications of the build machines [here]
+(https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners).
+
+For Unity projects the disc size is quite small. You may experience an error related to running out of disc space. You may also want to run the build on a server with more memory or CPU resources.
+
+**You can use Remote Builder to send a build from any supported git platform to any of the supported remote builder types.**
```

Comment:
```suggestion
**You can use remote builder to send a build from any supported git platform to any of the supported remote builder types.**
```

Either this is a keyword (in which case we can use caps) or its lower case. Looking at the title and no introduction of the term we do not intend a special keyword here right?